common/game_concepts/00_game_concepts.txt

```diff
@@ -986,6 +986,23 @@ heavy_infantry = {
 	parent = men_at_arms
```

Comment:
No icon.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/103152922-093bf680-47a6-11eb-8a71-72c20aa2b085.png)

</details>